docs/third-party-packages.md

```diff
@@ -23,3 +23,13 @@ An asynchronous GitHub API library. Includes [HTTPX support](https://gidgethub.r
 [GitHub](https://github.com/lundberg/respx) - [Documentation](https://lundberg.github.io/respx/)
 
 A utility for mocking out the Python HTTPX library.
+
+## Gists
+
+<!-- NOTE: this list is in alphabetical order. -->
+
+### urllib3-transport
```

Comment:
Not too keen to put this under "Plugins", since it's not really an installable package, but then also not keen on the existing "Plugins" section — perhaps we ought to just list everything as h2-level headers. We can address this as a follow-up.